htmlslacker/htmlslacker.py

```diff
@@ -72,6 +81,8 @@ def handle_endtag(self, tag):
             self.output += '`'
         if tag == 'style' or tag == 'script':
             self.skip = False
+        if tag == 'ol':
```

Comment:
There is no `ul` tag.

Reply:
based on the other code here, I'm not sure a `ul` is needed -- i'm not sure what it would do here.